publication-commons/src/main/java/no/unit/nva/publication/service/impl/MessageService.java

```diff
@@ -85,6 +97,28 @@ public void deleteMessage(UserInstance userInstance, Message message) throws Una
         getClient().transactWriteItems(transactionRequest);
     }
 
+    private boolean canManageMessage(Message message, UserInstance userInstance) throws NotFoundException {
+        if (userInstance.isSender(message)) {
+            return true;
+        }
+
+        return isCuratorForMessage(message, userInstance);
+    }
+
+    private boolean isCuratorForMessage(Message message, UserInstance userInstance) throws NotFoundException {
+        if (!message.getCustomerId().equals(userInstance.getCustomerId())) {
```

Comment:
This implies that we ignore fine-grained restrictions based on supposed hierarchies within an organisation, which for me is fine

Reply:
Yes, that is intentional 👍